korgan/payment_delivery_bridge.py

```
@@ -0,0 +1,55 @@
+            return await original(self, document, *args, **kwargs)
+
+        bot = self.bot
+        if bot is None:
+            LOGGER.error("PAYMENT_DELIVERY_NO_BOT kind=%s chat=%s", kind, getattr(self.chat, "id", None))
+            return await original(self, document, *args, **kwargs)
+
+        LOGGER.info("PAYMENT_DELIVERY_ROUTE_GATE user=%s kind=%s", self.chat.id, kind)
+        return await bot.send_document(
+            chat_id=self.chat.id,
+            document=document,
+            *args,
```

Ruff (0.16.2)

[warning] Star-arg unpacking after a keyword argument is strongly discouraged

(B026)